.idea/Toy_app.iml

```diff
@@ -40,6 +40,7 @@
     <orderEntry type="library" scope="PROVIDED" name="addressable (v2.7.0, ruby-2.7.2-p137) [gem]" level="application" />
```

Comment:
Please, add `.idea` to `.gitignore` because we don't need to commit `.idea` files.

Reply:
Done, thank You.